backend/LexCore/Entities/MediaFile.cs

```diff
@@ -0,0 +1,98 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace LexCore.Entities;
+
+public class MediaFile : EntityBase
+{
+    // Id property comes from EntityBase
+    public required string Filename { get; set; }
+    public Guid ProjectId { get; set; } // required; all files must belong to a project
+    public DateTimeOffset LastModified { get; set; } // You know what? We *should* derive from EntityBase after all.
```

Comment:
let's remove these comments, some aren't relevant or are just documenting how something else works

Reply:
Done in commit 35e3c6083, which also removed the completely-unused LastModified column (we already have UpdatedDate, and so I wasn't updating the LastModified column anyway). If we decide that LastModified should hvae different semantics than UpdatedDate, e.g. tracking the modification date of the file on the filesystem while UpdatedDate tracks the modified date of the MediaFile object, then we can do that. But at the moment I wasn't doing *anything* with LastModified, so I removed it completely.